rbac/providers/azure/delta_outbound_sync.py

```diff
@@ -162,22 +165,35 @@ def create_entry_aad(queue_entry):
 
 
 def create_user_aad(queue_entry):
-    """Creates a given user in aad."""
-    # TODO: Implement user creation in Azure AD. Currently logs and deletes
-    #   entry and throws an ExpectedError.
-    LOGGER.warning(
-        "User not in Azure AD. Aborting sync for user and removing \
-        from queue_outbound: %s",
-        queue_entry,
-    )
-    delete_entry_queue(queue_entry["id"], OUTBOUND_QUEUE)
-    raise ExpectedError("User not in Azure AD.")
+    """Creates a given user in AAD."""
+    headers = AUTH.check_token("POST")
+    if headers:
+        url = f"{GRAPH_URL}/{GRAPH_VERSION}/users"
+        try:
+            aad_user = outbound_user_creation_filter(queue_entry["data"], "azure")
+        except ValueError:
+            LOGGER.warning(
+                "Unable to create user in AAD, displayName and email required: %s",
+                queue_entry,
+            )
+            raise ExpectedError("Unable to create user without display name and email.")
+        aad_user["passwordProfile"] = {
+            "password": str(uuid4())[:16],
+            "forceChangePasswordNextSignIn": True,
+        }
+        response = requests.post(url=url, headers=headers, json=aad_user)
+        if response.status_code == 201:
+            delete_entry_queue(queue_entry["id"], OUTBOUND_QUEUE)
+        else:
+            LOGGER.warning("Unable to create user in AAD: %s", queue_entry)
+            raise ExpectedError("Unable to create user.")
 
 
 def create_group_aad(queue_entry):
     """Creates a given group in aad."""
     # TODO: Implement group creation in Azure AD. Currently logs and deletes
     #   entry and throws an ExpectedError.
+    LOGGER.info("You are in create_group_aad")
```

Comment:
Would this log be better in debug? As we are trying to not spam our logging output.

Reply:
Sorry I am using that for the next PR for testing.  I will remove.